timesketch/tsctl.py

```diff
@@ -804,33 +804,62 @@ def validate_context_links_conf(path):
 @cli.command(name="searchindex-info")
 @click.option(
     "--searchindex_id",
-    required=True,
-    help="Searchindex ID to search for e.g. 4c5afdf60c6e49499801368b7f238353.",
+    required=False,
+    help="Searchindex ID to search for e.g. 3.",
+)
+@click.option(
+    "--index_name",
+    required=False,
+    help="Searchindex name to search for e.g. 4c5afdf60c6e49499801368b7f238353.",
 )
-def searchindex_info(searchindex_id: str):
+def searchindex_info(searchindex_id: int, index_name: str):
     """Search for a searchindex and print information about it.
-    Especially which sketch the searchindex belongs to.
+    Especially which sketch the searchindex belongs to. You can either use the
+    searchindex ID or the index name.
+
 
     Args:
-        searchindex_id (str): The search index ID to search for (e.g.,
+        searchindex_id (int): The search index ID to search for (e.g.,
+                              "3").
+        index_name (str): The search index ID to search for (e.g.,
                               "4c5afdf60c6e49499801368b7f238353").
     """
+    if searchindex_id:
+        if not searchindex_id.isdigit():
+            print("Searchindex ID needs to be an integer.")
+            return
 
+    index_to_search = None
     index_to_search = SearchIndex.query.filter_by(index_name=searchindex_id).first()
```

Comment:
I don't think this will work? `index_name=searchindex_id`. Do you need this here at all?